pkgs/development/python-modules/embrace/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, sqlparse, wrapt }:
+
+buildPythonPackage rec {
+  pname = "embrace";
+  version = "4.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-hKxXVwenf9EU6YUNS6xzSDW9VvVSlewakyd8Zse+kNs=";
+  };
+
+  # embrace does not include its tests in the pypi dist archive
+  doCheck = false;
```

Comment:
The upstream source contain [tests](https://hg.sr.ht/~olly/embrace-sql/browse/tests?rev=tip). Consider using [`fetchFromSourcehut`](https://nixos.org/manual/nixpkgs/stable/#fetchfromsourcehut) to get the source.

Reply:
Amended to use the upstream sources and enable the tests :+1: